api/src/main/java/org/eclipse/microprofile/openapi/annotations/security/SecurityRequirement.java

```diff
@@ -48,10 +48,9 @@
 
     /**
      * If the security scheme referred by the name property is of type "oauth2" or "openIdConnect", then the scopes
-     * array is a list of scope names required for the execution.
-     * <p>
-     * For other security scheme types, the array MUST be empty.
-     * </p>
+     * array is a list of scope names required for the execution, and the list MAY be empty if authorization does not
+     * require a specified scope. For other security scheme types, the array MAY contain a list of role names which are
+     * required for the execution, but are not otherwise defined or exchanged in-band.
```

Comment:
This is copied from the OpenAPI spec (Apache 2.0 license) - need to check if we're ok to include this as-is.

Reply:
Didn't find this out so I'm going to redo this change, starting from the original javadoc.